### DISCUSSION TITLE
Icloud File Icon

### DISCUSSION THREAD

**Discussion by underthestars-zhy** [elrumo/macOS_Big_Sur_icons_replacements#1338] [Icon request]
This icon is widely used in files app. But I can't find it in the website
![IMG_6913](https://user-images.githubusercontent.com/46642225/145380538-5d4a9db4-302d-4699-bbdd-309c6439f408.jpg)
.

**Comment by Teichman22** [comment]
> This icon is widely used in files app. But I can't find it in the website
> 
> ![IMG_6913](https://user-images.githubusercontent.com/46642225/145380538-5d4a9db4-302d-4699-bbdd-309c6439f408.jpg)
> 
> .

I just remade these for you.

https://macosicons.com/u/teichman
